guardian/fsbp-tools
# Run dash-lint on ingress-inquisition

## What does this change?

Lints the project for instances of `ingress-inquisition` without dashes, by making the original dash-lint.sh script reusable

## How to test

Introduce an `ingressinquisition` comment somewhere in the codebase, then run `./script/dash-lint.sh ingress-inquisition`. The script should fail. Remove it, and it should pass. CI should pass
